packages/node-runtime-worker-thread/src/index.ts

```diff
@@ -70,6 +74,12 @@ class WorkerRuntime implements Runtime {
       }
     });
 
+    // We expect the amount of listeners to be more than the default value of 10
+    // but probably not more than ~15 (all exposed methods on
+    // ChildProcessEvaluationListener and ChildProcessMongoshBus + any
+    // concurrent in-flight calls on ChildProcessRuntime) at once
+    this.childProcess.setMaxListeners(15);
```

Comment:
I mean, I think setting these to `Infinity` is fine, unless the warning provides some value to us

Reply:
Did it on the off chance something will be actually messed up and too many handlers will be attached